2016/proposals/result-aggregation-server-garg-prerit.md

```diff
@@ -0,0 +1,404 @@
+
+* My skill sets include : C/C++, Python - Flask, Django, Java, Android, HTML, CSS, JavaScript, MySQL, Sqlite, NoSQL - [MongoDB](https://www.mongodb.org/), REDIS, [ANSIBLE](https://www.ansible.com/).
+ 
+* I have mostly worked with Web Development ( RESTful API's, Dashboards etc ) while working as an intern with Zopper. I worked with Google Cloud Messaging ( GCM ) to send notification and bulk notification to more than 1 million zopper application users.  
+ 
+* I also have experience in Android Application Development. I developed an Android Application for [Litchi](http://www.litchi.co.in/), published it on PlayStore. and also a demo [FourSquare](https://github.com/prerit2010/FourSquare) application to gain more experience in android development. 
+ 
+* Apart from this, as the academic project I worked on cryptography, and published a research paper on "Efficient Substitution Box Design using Travelling Salesman Problem and Chaos". 
+ 
+## Why this project?
+This project is concerned with the problems that students face in installing the necessary packages for a workshop. This project will enable the instructors to better understand the problems and remove the package version and other issues, so that the setup process speeds up, and students don't get stuck in just installing the packages for a long time. <br />
+I have a lot of experience in developing RESTful API's and web services. This project involves development of a RESTful API and setting up a server. Working upon this, it would give me more experience towards working on APIs and python Scripts, and also a open source experience which is new and exciting for me.
+ 
+## Appendix
+
+Link to my opened issues : [#87](https://github.com/numfocus/gsoc/issues/87) , [#82](https://github.com/numfocus/gsoc/issues/82)
+
+### Operating systems accessibility 
+
+* **Laptop configuration** : Lenovo Z510 with Ubuntu 15.10 and Windows 10.
+* I shall be testing the scripts on all three platforms ( Linux , Windows , OS X ). I have Linux and Windows on my Laptop, and also have access to OS X through two of my close friends' MacBook.
+
+### Code Contribution
+* As per the requirement mentioned in the project information, I developed a sample [demo](https://github.com/prerit2010/DemoAPI) flask API which takes the Operating system information and sends to the API making a POST request.
+* I also created a [sample django application](https://github.com/prerit2010/SurveyMonkey-Sample) using SurveyMonkey API and displaying the plots using plotly.
+
+### Availability
+* **Time Zone :** Indian Standard Time (IST) UTC +5:30 
+* 2nd Mid semester examinations start: 9th May, 2016
+* 2nd Mid semester examinations end : 23rd May, 2016 (Approx.)
+* My graduation (B.Tech) completes as soon as exams get over. So, after that I am available full time for the project.
```

Comment:
@rgaiacs No, I'll be appearing for [GATE](https://en.wikipedia.org/wiki/Graduate_Aptitude_Test_in_Engineering) exam next year, so will be preparing for that.